### ISSUE TITLE
Create a validator that need to be done before saving into OSM

### ISSUE THREAD

**Issue by meomancer** [healthsites/healthsites#1129]
# Problem
Some of fields of osm just be able to receive some values, so we need to check it.

# Solution
Validator is needed to check some values of field. We could update this validator maybe in json. If we use json, we could also update the schema (ticket : https://github.com/healthsites/healthsites/issues/1128) and this validator can use this schema

Here are the validation:
1. **mandatory** : [amenity, healthcare, name, operator, source]
2. **dispensing** turn to be mandatory if amenity is **pharmacy**
3. **amenity** options are : [clinic, doctors, hospital, dentist, pharmacy]
4. **Healthcare** options are : [doctor, pharmacy, hospital, clinic, dentist, physiotherapist, alternative, laboratory, optometrist, rehabilitation, blood_donation, birthing_center]
5. **operator_type** options are : [public, private, community, religious, government, ngo, combination]
6. **operational_status** options are : [operational, non_operational, unknown]
7. **health_amenity_type** options are : [ultrasound, mri, x_ray, dialysis, operating_theater, laboratory, imaging_equipment, intensive_care_unit, emergency_department]
8. **dispensing,wheelchair,emergency** are boolean
9. **insurance_health** options are [no, public, private, unknown]
10. **water_source** options are [well, water_works, manual_pump, powered_pump, groundwater, rain]
12. **electricity** options are [grid, generator, solar, other, none]

13. **speciality** is an unique check
please check this:
https://wiki.openstreetmap.org/wiki/Key:healthcare#Subtags
speciality can has those values: (e.g: allergology, anatomy, etc)
but if **healthcare** is clinic, the options will be [abortion, fertility] (please see "Specialities for healthcare=clinic" section on the wiki)

**Comment by meomancer** [comment]
Already done by @myarjunar
